pallets/gear/src/schedule.rs

```diff
@@ -36,10 +37,11 @@ use sp_std::{marker::PhantomData, vec::Vec};
 /// How many API calls are executed in a single batch. The reason for increasing the amount
 /// of API calls in batches (per benchmark component increase) is so that the linear regression
 /// has an easier time determining the contribution of that component.
-pub const API_BENCHMARK_BATCH_SIZE: u32 = 100;
+pub const API_BENCHMARK_BATCH_SIZE: u32 = 80;
```

Comment:
why do we dump these amounts of calls?

Reply:
What about to go with 20 / 50 as is done in pallet contracts?

Reply:
> What about to go with 20 / 50 as is done in pallet contracts?

They have 80/100 in constants. 100 for instructions don't have time to warm up lazy pages.